pkgs/development/python-modules/gpg/default.nix

```diff
@@ -1,18 +1,13 @@
-{ stdenv, pythonPackages
+{ stdenv, buildPythonPackage
 , fetchPypi, libgpgerror, swig2
 , gpgme
 , withManpage ? true }:
 
-with pythonPackages;
-
-
-
```

Comment:
This change uncovers that the library still fails to build on python3. But I have not investigated the reason for that.